src/styles/app/23-node-note-editor-a.css

```
@@ -241,8 +241,8 @@
 	width: min(
 		100%,
 		calc(
-			var(--readable-content-max-width) +
-			(2 * var(--readable-content-gutter-compact))
+			var(--editor-readable-content-max-width) +
```

Stylelint (17.7.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)